modules/ROOT/pages/test-recorder.adoc

```diff
@@ -5,10 +5,41 @@ endif::[]
 
 The test recorder enables you to record a processing flow and then configure a unit test based on the captured event.
 
-The test recorder is an open beta feature that is compatible with Anypoint Studio 7.x.x and later and MUnit 2.2.1 and later.
+The test recorder captures real data as it goes through your application in Studio. By capturing the flow execution, MUnit can automatically generate an MUnit test so that you can configure the necessary mocks and assertions for your test. See xref:test-recorder-examples.adoc[Recording a Test from an Exchange Example] for a full working example.
+
+[NOTE]
+The test recorder is an open beta feature.
+
+== Compatibility
+
+The following table describes the required product versions to use the test recorder. These versions are for the beta feature and are likely to change for the GA (general availability) release.
+
+[%header%autowidth.spread,cols="a,a"]
+|===
+| Product | Version
+| Anypoint Studio
+| 7.4.0 and later.
+
+| MUnit
+| 2.2.4
+
+| Mule runtime engine
+| Patched version of 4.2.2. +
+See xref:runtime-patching.adoc[Patching the Embedded Mule Runtime Engine Version] to learn how to apply this patched version to your project.
```

Comment:
This is not how you apply this patched version. That defines the version used while running the MUnit tests, but the recorder happens during the app execution, not the test execution. What you need to do is to go to `Install new software`, in the `Work with` drop down menu select `Anypoint Studio Enterprise Runtimes`, and then select `Mule Server Runtime 4.2.2 EE`. This should install the latest 4.2.2 runtime available.

Reply:
Got it. Wrong link then.